server/models/Request.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+    {
+        ownerId: {
+            type: mongoose.Schema.Types.ObjectId,
+            required: true,
+            ref: 'User'
+        },
+        ownerName: {
```

Comment:
Let's remove a few fields from this model as we wouldn't want to duplicate data across our db. Please see below for fields to be removed:
- ownerName
- ownerPhoto
- sitterName
- location